mathics/builtin/numbers/calculus.py

```diff
@@ -577,6 +587,8 @@ def apply(self, f, n, n0, evaluation, options={}):
 
 class _BaseFinder(Builtin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/_BaseFinder.html</url>
```

Comment:
This link is broken

Reply:
This comes from the automatic fixing, and should be removed.